src/test/suite/fixtures/recorded/actions/pasteAfterLineSpunAndAfterBlockLookAndBeforeLineSpun.yml

```diff
@@ -0,0 +1,82 @@
+languageId: plaintext
+command:
+  spokenForm: paste after line spun and after block look and before line spun
```

Comment:
Shockingly, Cursorless handled this one with no problem, inserting the right delimiters and leaving cursors in the right spot 😯